make_plant_parts_eia.py

```diff
@@ -1,4 +1,57 @@
-"""Aggregate plant parts."""
+"""
+Aggregate plant parts to make an EIA master unit list.
+
+The EIA data about power plants (from EIA 923 and 860) is reported in tables
+with records that correspond to mostly generators and plants. Practically
+speaking, a plant is a collection of generator(s). There are many attributes
+of generators (i.e. prime mover, primary fuel source, technology type). We can
+use these generator-based attributes to create other records about other
+collections of plant parts. A "plant part" record which cooresponds to a
+particular collection of generator attributes (i.e. all of the generators with
+unit id "2").
+
+Because generators are often owned by multiple utilities, another dimention of
+the master unit list involves generating two records for each owner: one of the
+portion of the plant they own and one for the plant as a whole. The portion
+records are labeled in the `ownership` column as "owned" and the total records
+are labeled as "total".
+
+This module refers to "true granularies". Many plant parts we cobble together
+here in the master unit list refer to the same collection of infrastructure as
+other master unit list records. For example, if we have a "plant_prime_mover"
+plant part record and a "plant_unit" plant part record which were both cobbled
+together from the same two generators. We want to be able to reduce the master
+unit list to only unique collections of generators, so we label the first
+unique granularity as a true granularity and label the subsequent records as
+false granularities with the `true_gran` column. In order to choose which plant
+part to keep in these instances, we assigned a `plant_parts_ordered` and
+effectively keep the first instance of a unique granularity.
```

Comment:
However, we want each record in the master unit list to correspond to a unique collection of generators. To remove duplicate collections of generators associated with different plant parts, we have assigned an ordering to the list plant parts, from highest to lowest priority, and we retain only the highest priority plant part record in which a given collection of generators appears. This highest priority plant part is referred to as the record's "true granularity" and all other collections of the same generators are referred to as "false granularities"